src/main/redux/sagas/app.ts

```diff
@@ -78,7 +78,7 @@ export function exit() {
         const shutdownEventChannel = getShutdownEventChannel();
         const windowAllClosedEventChannel = getWindowAllClosedEventChannel();
         const quitEventChannel = getQuitEventChannel();
-        let shouldExit: boolean = process.platform !== "darwin" || IS_DEV;
+        let shouldExit = process.platform !== "darwin" || IS_DEV;
```

Comment:
shall we remove the dev condition here, so that prod and dev use the exact same teardown logic?

Reply:
i just a preference to kill app in dev. If the app stay in dock i should make a CTRL-C in dev console. In dev we make many re-start.  Are you agree ?

Reply:
ok